src/ui/Windows/ThisMfcApp.h

```diff
@@ -42,6 +42,15 @@ class DboxMain;
 
 class ThisMfcApp : public CWinApp
 {
+public:
+  enum AllowScreenCaptureState
```

Comment:
I keep it in place for the latest revision pending our resolving the final remaining issue... generic message for any/all of RDP/ICA, or specific message.

Since we already have plumbing in place indicating a non-general specific message for each case, my vote is we retain such as it's more helpful without diminishing UX. If we keep that detail in the messages, I think the enum or something like it reflecting the state detail must be in place.